### ISSUE TITLE
PDF not showing GCODE correct Print time

### ISSUE THREAD

**Issue by Antonio36996** [AndreasReitberger/3D-Print-Cost-Calculator#11]
Hi! I noticed that the Print time shown in the overview section is not the same as the shown in the gcode section, which is the correct one.

![image](https://github.com/AndreasReitberger/3D-Print-Cost-Calculator/assets/66578721/58ba2337-7025-4082-8fb0-8dc9358df435)
![image](https://github.com/AndreasReitberger/3D-Print-Cost-Calculator/assets/66578721/bab6fa65-7e10-4e1c-9b9e-4cfcf4a3a52a)

**Comment by AndreasReitberger** [comment]
Hi @Antonio36996 
the print time in the overview is the total print time (with fail rate time added). However I'll double check this.

**Comment by AndreasReitberger** [comment]
I checked the source code, the value is accurate, however confusing. So I adjusted the pdf export to also show the additional time (if a fail rate is defined > 0)

Also now both times are shown in the same format.
![image](https://github.com/AndreasReitberger/3D-Print-Cost-Calculator/assets/2952443/d87751ca-84b5-45b2-b892-049695c32682)

![image](https://github.com/AndreasReitberger/3D-Print-Cost-Calculator/assets/2952443/bcac4abd-bc8d-4610-bd62-5da11935b313)

This fix will be available in 1.4.5. Thank you!
